packages/webpack-cli/lib/plugins/WebpackCLIPlugin.js

```diff
@@ -1,62 +1,143 @@
 const { packageExists } = require('../utils/package-exists');
 const webpack = packageExists('webpack') ? require('webpack') : undefined;
-const logger = require('../utils/logger');
+const { getStatsOptions } = require('../utils/stats-options');
+const { PluginName } = require('../utils/name');
 
-const PluginName = 'webpack-cli';
-
-class WebpackCLIPlugin {
-    constructor(options) {
-        this.options = options;
+class WebpackCLIRuntimePlugin {
```

Comment:
No need to rename, keep WebpackCLIPLugin

Reply:
Yep, just was playing around with some ideas. Would be nice if `webpack-dev-middleware` and `webpack-cli` output messages in same format. Basic sharing the plugin https://github.com/webpack/webpack-dev-middleware/blob/master/src/utils/setupHooks.js